corehq/apps/export/tests/test_export_item.py

```diff
@@ -54,3 +55,19 @@ def test_wrap_export_item_child(self):
         wrapped = ExportItem.wrap(item.to_json())
         self.assertEqual(type(wrapped), type(item))
         self.assertEqual(wrapped.to_json(), item.to_json())
+
+
+def TestGeopointItem(SimpleTestCase):
+
+    def test_split_value(self):
```

Comment:
Could make this more concise by using [`generate_cases`](https://github.com/dimagi/commcare-hq/blob/master/corehq/util/test_utils.py#L259)